src/pytds/tds_base.py

```diff
@@ -955,7 +956,7 @@ def __init__(self) -> None:
         self.validate_host = True
         self.enc_login_only = False
         self.enc_flag = 0
-        self.tls_ctx = None
+        self.tls_ctx: None | OpenSSL.SSL.Context = None
```

Comment:
allowing pipes for Union types doesn't work in python 3.9: https://peps.python.org/pep-0604/

Reply:
Does this mean 1.16.0 will not work in python 3.9, or this just affects the package building?

Reply:
I see. Thanks

Reply:
This module has
```python
from __future__ import annotations
```
Which apparently enables Union type support for Python starting at least from 3.8

For example this code works on Python 3.8:
```python
from __future__ import annotations
class Foo:
    asdf: None | int
```